google/cloud/aiplatform/experimental/vertex_model/base.py

```diff
@@ -234,23 +243,68 @@ def f(*args, **kwargs):
                 + SERVING_COMMAND_STRING_CODE_APIS
             )
 
+            # Account for user-designated dependencies when
+            # setting up remote prediction
             if GITHUB_DEPENDENCY not in obj.dependencies:
                 obj.dependencies.append(GITHUB_DEPENDENCY)
 
+            dependency_installs = []
+            for dependency in obj.dependencies:
+                if dependency != GITHUB_DEPENDENCY:
+                    dependency_name = re.split(">|=|<", dependency)[0]
```

Comment:
It seems like we should keep the version of the dependency.

Reply:
I split around the version here to make the pip install easier. How would I parse the string to install a specific version of each library? I'll look into it right now as well.

Reply:
You should be able to pass the dependency definition name as-is without splitting.

Reply:
okay, will make this change right now and test it out.